normandy/classifier/api.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers, views, status
+from rest_framework.response import Response
+
+from normandy.bundler.serializers import BundleSerializer
+from normandy.classifier.models import Client
+from normandy.bundler.models import Bundle
+
+
+class FetchBundle(views.APIView):
+
+    serializer_class = BundleSerializer
+
+    class Parameters(serializers.Serializer):
```

Comment:
This isn't a standard DRF feature. I just made this up today. Well, I've used serializers for API parameters before, but this nested class is new. I used it by hand in the `post` method below, but in the future it is probably worth making a mixin that does it automatically.